sdk/identity/azure-identity/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.27.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <version>1.27.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core-test;dependency} -->
```

Comment:
You may need unreleased_com.azure:azure-core as well

Reply:
I did, fixed.